src/main/java/racingCar/RacingCar.java

```diff
@@ -15,11 +17,20 @@ public void tryMove(int number) {
     this.moveCount++;
   }
 
+  private boolean canMove(int randomNumber) {
+    return randomNumber >= 4;
+  }
+
   public int getMoveCount() {
     return moveCount;
   }
 
-  private boolean canMove(int randomNumber) {
-    return randomNumber >= 4;
+  public String getName() {
+    return name;
+  }
+
+  @Override
+  public int compareTo(RacingCar racingCar) {
+    return moveCount - racingCar.getMoveCount();
   }
```

Comment:
Comparable 구현 👍 

```suggestion
  @Override
  public int compareTo(RacingCar racingCar) {
    return moveCount - racingCar.moveCount;
  }
```

getter를 사용하지 않고도 가능하겠네요! 
moveCount, name 또한 포장해볼수 있지 않을까요?